src/app/app-reservation-action-page/app-reservation-action-page.component.scss

```diff
@@ -0,0 +1,114 @@
+.empty-card {
+    max-width: 80%;
+    margin: auto;
+  }
+  
+  .mat-standard-chip {    
+    height: auto !important;
+  }
+  
+  .loading-spinner{
+    margin:auto;
+    max-width: 40%;
+    padding-top: 20px;
+  }
+  .container {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    padding-top: 10px;
+    margin-left: 10px;
+    margin-right: 10px;
+
+    @media (max-width: 500px) { 
+        margin-left: 0px;
+        margin-right: 0px;
+    }
+
+    .main-content {
+        flex: 1;
+        max-width: 1000px;
+    }
+}
+    .main-container-reservations {
+        display: flex;
+        justify-content: center;
+        position: relative;
+
+        @media (max-width: 500px) { 
+            flex-direction: column;
+            }
+  
+        mat-divider{
+            bottom: 0;
+        }
+        
+        .card-content {
+            display:flex;
+            width: 80%;
+            margin-top: 20px;
+            //background: hsla(0,0%,100%,.1);
+
+            @media (max-width: 500px) { 
+            flex-direction: column;
+            width: 100%;
+            }
+
+            .itemBox-imgbox{
```

Comment:
hoofdletters mogen niet in SCSS (of css in het algemeen) classes. 
Dit wordt dus `item-box-imgbox` (of anders)
Vergeet niet dan ook de HTML aan te passen :p